copter/source/docs/aerobatic-multicopter.rst

```diff
@@ -0,0 +1,42 @@
+.. _aerobatic-multicopter:
+
+============================
+Aerobatic/Racing Multicopter
+============================
+
+**this page is a work in progress and is incomplete currently**
+
+With the proper vehicle configuration and careful tuning, a precise, aggressive multicopter can be obtained for racing or aerobatics in ACRO mode.
+
+The first requirement is a vehicle frame that is very stiff to resist resonant vibrations and a power system that delivers high thrust to weight ratios. This is most easily obtained with smaller (< 6inch diagonal motor distances) frames.
+
+Obviously, racing miniquad frames already meet these requirements, usually. The advantage of using ArduPilot firmware is the ability to combine aggressive flight characteristics with autonomous features like FOLLOW mode, and RTL mode. Even :ref:`object avoidance <common-simple-object-avoidance>` can be included in some cases.
+
+Setup
+=====
+
+Follow the normal ArduCopter wiki :ref:`setup guidlines <configuring-hardware>`. However, the key to getting the best performance will be determined by the care and precision of the :ref:`tuning and noise filtering setup<common-tuning>`. And key to getting highest performance from the filtering will be use of Bi-Directional DSHOT ESCs to provide individual motor rpm for setting individual notch filers to track each motor (see :ref:`common-esc-telemetry`.
+
+tbd...video on notch filter setup
+
+tbd...video on tweaking tuning for higher aggression
+
+GPS needed?
+===========
+
+You do not need a GPS to fly in many Copter modes (ACRO, STABILIZE, ALTHOLD,etc.)
+
+
+Consequences of Aggressive Flight with GPS
+==========================================
+
+If you do include a GPS for the advance mode capabilities, then there are some potential consequences that can occur when switching to position holding/tracking modes after aggressive maneuvering in modes like ACRO. These are:
+
+- :ref:`EKF Failsafes<ekf-inav-failsafe>` occurring
+- Large position errors, ie like switching to RTL and then overshooting HOME position by very large amount, or tracking waypoints in a mission with large positioning errors
+
+Both of these are due to errors generated by high accelerations resulting in GPS miss-measurement of velocities, resulting in the navigation filter declaring itself unhealthy (EKF Failsafe), or very bad position estimates.
+
+These issues CAN be reduced/eliminated by either setting the :ref:`EK3_GLITCH_RAD<EK3_GLITCH_RAD>` = 0 , or increasing :ref:`FS_EKF_THRESHOLD<FS_EKF_THRESHOLD>`, or both. This should be used only if experiencing the above problems since it can degrade positioning accuracy rather then improve it in some cases, or allow EKF degradation to occur to the point of possible vehicle instability.
+
```

Comment:
Need a section on learning gyro biases and heat. Lowering EK3_ABIAS_P_NSE and EK3_GBIAS_P_NSE will prevent the the build-up of vibration related biases, but it needs to be coupled with a good TCAL setup to account for the heat related biases that occur. If you let the biases build up you will find the copter's trim is way off when switching back to EKF modes.

Reply:
what mode doesnt have EKF? do you mean to STABILIZE type modes from ACRO? or to positioning modes from non-position dependent modes

Reply:
Essentially only Acro doesn't use the EKF at all, and that's only if you've selected the gyro-only mode, although even if you don't have it selected, the EKF doesn't affect it too much.
And how well the ABIAS/TCAL method works likely also depends on the IMUs you have - with an MPU6000 it may not work very well compared to the newer IMUs. Also interesting 'cause I don't think I've seen this accelbias issue on Braveheart. Might depend on the specific vehicle setup.